core/commands/checkout.py

```diff
@@ -37,7 +37,12 @@ def on_branch_selection(self, branch):
 
         self.git("checkout", branch)
         sublime.status_message("Checked out `{}` branch.".format(branch))
-        util.view.refresh_gitsavvy(self.window.active_view(), refresh_sidebar=True)
+        refresh_all_visible_views(self.window)
```

Comment:
Move and rename to 
`util.view.refresh_all_gitsavvy`

Reply:
Done.